apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -128,7 +134,7 @@ function ValidatorPageResult(): JSX.Element {
         : [];
 
     const tableColumns = useMemo(() => {
-        if (!data || !validatorEvents) return null;
+        if (!data || !protocolConfig || !validatorEvents) return null;
         const includeColumns = [
```

Comment:
can you update the PlaceholderTable to keep consistence and avoid table jumps? :pray: 
![Screenshot from 2025-04-28 09-12-44](https://github.com/user-attachments/assets/09b3c1ac-5c7b-4991-99fd-a263ce445948)

Reply:
[1f7d88f](https://github.com/iotaledger/iota/pull/6601/commits/1f7d88f8da31e3b737e9b9e36e1885d592d381cb)